dotnetflix.Api/Data/DbContext.cs

```diff
@@ -85,10 +86,51 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                     );
                 }
             }
-        }
+
+			// Seed data for Seat
+
+			/* De bioscoop bestaat uit 6 zalen: 3 grote zalen (1 tot en met 3) en 3 kleine (4 tot en met 6).
```

Comment:
Deze comments zijn onnodig, dit staat in de requirements en willen we niet in de code hebben.